src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days < 3) {
+    total = 40 * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
Вместо того чтобы проверять противоположное предыдущему условие обычно ставят else if
```suggestion
  if (days < 3) {
    total = 40 * days;
  } else if (days < 7) {
```